example/helm/values-scylla.yaml

```
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+agent:
+  minReplicas: 5
+  maxReplicas: 10
+  podManagementPolicy: Parallel
+  ngt:
+    dimension: 784
```

it should be override default resource requests like belows:

```suggestion
gateway:
  resources:
    requests:
      cpu: 100m
      memory: 50Mi

agent:
  minReplicas: 5
  maxReplicas: 10
  podManagementPolicy: Parallel
  resources:
    requests:
      cpu: 100m
      memory: 50Mi
  ngt:
    dimension: 784

discoverer:
  resources:
    requests:
      cpu: 100m
      memory: 50Mi

compressor:
  resources:
    requests:
      cpu: 100m
      memory: 50Mi

indexManager:
  resources:
    requests:
      cpu: 100m
      memory: 30Mi
```